src/models/observations.js

```diff
@@ -1,19 +1,59 @@
 // @flow
 import type { Observation } from '../types/observation';
+import type { ObservationAPI } from '../api/observations';
 
-export const applyObservationDefaults = (partial: Object): Observation => ({
-  type: 'Rios y corrientes',
+export const defaultObservation: Observation = {
+  id: '',
   lat: 0,
   lon: 0,
-  link: 'link',
   created: new Date(),
   name: '',
   notes: '',
   observedBy: 'Tú',
   attachments: [],
-  icon: null,
-  categoryId: '0',
-  fields: [],
+  categoryId: '',
+  fields: []
+};
 
-  ...partial
-});
+export const parseObservationRequest = (
+  observation: Object
+): ObservationAPI => {
+  const { lat, lon, attachments, version } = observation;
+  const tags = { ...observation };
+
+  // remove observation defaults
+  const keep = { created: true };
+  Object.keys(tags).forEach(property => {
+    if (!keep[property] && tags[property] === defaultObservation[property]) {
+      delete tags[property];
+    }
+  });
+
+  // remove topLevel properties
+  const topLevel = ['lat', 'lon', 'attachments', 'id'];
+  topLevel.forEach(p => delete tags[p]);
```

Comment:
This is where we separate the toplevel properties from those we're keeping in tags. The property in tags are essentially everything that isn't top level and in the observation model, ie: created, categoryId, fields, etc.